src/ontology/mondo-edit.obo

```diff
@@ -245188,14 +245188,15 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/9946/arthrog
 id: MONDO:0012391
 name: neuronal ceroid lipofuscinosis 8 northern epilepsy variant
 def: "Progressive epilepsy-intellectual deficit, Finnish type (also known as Northern epilepsy) is a subtype of neuronal ceroid lipofuscinosis (NCL) characterized by seizures, progressive decline of intellectual capacities and variable loss of vision." [Orphanet:1947]
-subset: gard_rare
```

Comment:
Why is this removed?

Reply:
it is a duplicate, it's another external file too (there is an external mondo-subsets.owl file)

Reply:
Ok. What is it a duplicate of?

Reply:
the exact same annotation is in the mondo-subsets.owl file.

I think these were moved into the external file when @sabrina and @matentzn created the new rare disease subsets.

See: https://github.com/monarch-initiative/mondo/issues/6167

Reply:
Ok, I don't know what the `mondo-subsets.owl` file is.

Reply:
it is in mondo/components

it looks like we don't have any documentation on it, can you bring this up on the tech call? Nico can explain what it is, and we should create some documentation.

Reply:
Thank you! I added this to the tech call agenda.